installer/pkg/cmd/gcp_broker.go

```diff
@@ -260,6 +262,17 @@ func removeGCPBroker() error {
 		return fmt.Errorf("error deleting broker resources : %v", err)
 	}
 
+	projectID, err := gcp.GetConfigValue("core", "project")
+	if err != nil {
+		return fmt.Errorf("error getting configured project value : %v", err)
+	}
+
+	brokerSAEmail := fmt.Sprintf("%s@%s.iam.gserviceaccount.com", brokerSAName, projectID)
+	err = gcp.RemoveServiceAccountPerms(projectID, brokerSAEmail, brokerSARole)
```

Comment:
Is the role assigned in the context of a project?
If I have 2 clusters in my project, add broker to both, and then remove the broker from one ... what happens?

Reply:
Yeah, good catch. This is just like the broker resource, and is assigned in the context of a project. We should not remove the role unless we are sure all the clusters under the project have removed the gcp broker.

So we can't do it unless we have a counter associated with each project.

@mihnjong Could you provide more context with the original problem so we can workaround it?